README.md

```diff
@@ -161,6 +161,12 @@ It is highly recommended that you use Amazon S3. The reasons are:
 
 Why then not require S3? Because while some applications using node-pre-gyp need to distribute binaries as large as 20-30 MB, others might have very small binaries and might wish to store them in a GitHub repo. This is not recommended, but if an author really wants to host in a non-S3 location then it should be possible.
 
+It is also possible to use an s3-compatible storage backend, such as minio, as well as non-standard s3 endpoints. In this case, you can specify the following properties in the `binary` section:
+  - `host`: should be the root url of the storage server.
+  - `bucket`
+  - `region`
+  - `s3ForcePathStyle` [optional, default: `false`, must be set to `true` for minio servers]
```

Comment:
Could you try to make this comment more generic? For example, there might be users that need to set this to `true` but are not using or familiar with `minio`: how would they know to set it to `true` / can you describe more detail about why it needs to be set to `true`?

Reply:
How about:
Set s3ForcePathStyle to true if the endpoint url should not be prefixed with the bucket name. If false (default), the server endpoint would be contructed as `bucket_name.your_server.com`.

Reply:
👍